modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,45 +25,43 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
-pod/ovnkube-node-jg52r     5/5     Running   0             51m
-pod/ovnkube-node-k8wf7     5/5     Running   0             57m
-pod/ovnkube-node-tlwk6     5/5     Running   0             47m
-pod/ovnkube-node-xsvnk     5/5     Running   0             57m
+NAME                       READY   STATUS    RESTARTS        AGE
+pod/ovnkube-master-2qnbv   6/6     Running   0               19m
+pod/ovnkube-master-kqmnv   6/6     Running   1 (11m ago)     19m
+pod/ovnkube-master-rbrds   6/6     Running   0               19m
+pod/ovnkube-node-7xlcc     5/5     Running   1 (8m25s ago)   9m14s
+pod/ovnkube-node-7xshp     5/5     Running   1 (9m28s ago)   10m
+pod/ovnkube-node-cwrmk     5/5     Running   1 (18m ago)     19m
+pod/ovnkube-node-dxnlt     5/5     Running   1 (18m ago)     19m
+pod/ovnkube-node-jcdh6     5/5     Running   1 (8m50s ago)   9m33s
+pod/ovnkube-node-xrml8     5/5     Running   1 (18m ago)     19m
 
 NAME                            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)             AGE
-service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            57m
-service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   57m
-service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   57m
+service/ovn-kubernetes-master   ClusterIP   None         <none>        9102/TCP            19m
+service/ovn-kubernetes-node     ClusterIP   None         <none>        9103/TCP,9105/TCP   19m
+service/ovnkube-db              ClusterIP   None         <none>        9641/TCP,9642/TCP   19m
 
 NAME                            DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                                 AGE
-daemonset.apps/ovnkube-master   3         3         3       3            3           beta.kubernetes.io/os=linux,node-role.kubernetes.io/master=   57m
-daemonset.apps/ovnkube-node     5         5         5       5            5           beta.kubernetes.io/os=linux                                   57m
+daemonset.apps/ovnkube-master   3         3         3       3            3           beta.kubernetes.io/os=linux,node-role.kubernetes.io/master=   19m
+daemonset.apps/ovnkube-node     6         6         6       6            6           beta.kubernetes.io/os=linux                                   19m
 
-NAME                              ENDPOINTS                                                        AGE
-endpoints/ovn-kubernetes-master   10.0.132.11:9102,10.0.151.18:9102,10.0.192.45:9102               57m
-endpoints/ovn-kubernetes-node     10.0.132.11:9105,10.0.143.72:9105,10.0.151.18:9105 + 7 more...   57m
-endpoints/ovnkube-db              10.0.132.11:9642,10.0.151.18:9642,10.0.192.45:9642 + 3 more...   57m
+NAME                              ENDPOINTS                                               AGE
+endpoints/ovn-kubernetes-master   10.0.0.3:9102,10.0.0.4:9102,10.0.0.5:9102               19m
```

Comment:
I fail to understand why the outputs here are being changed.

Reply:
Changing back